zsh.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: workstation
+
+  tasks:
+    - name: get zsh script
+      get_url: url=http://github.com/robbyrussell/oh-my-zsh/raw/master/tools/install.sh dest=/tmp/zsh.sh
+
+    - name: change file permission to be executable
+      command: chmod +x /tmp/zsh.sh
+
+    - name: install zsh
```

Comment:
it's not installing zsh, it's installing `oh-my-zsh`, please change the names here.